docs/targets/tekton.md

```diff
@@ -1,85 +1,75 @@
 # Tekton Pipeline Event Target for Knative Eventing
 
-This event target integrates with Tekton pipelines using received Cloud Event
-messages to trigger a pipeline run or a task run.
+This event target receives [CloudEvents][ce] over HTTP and sends them to [Tekton][tekton]
+to invoke a Tekton build task.
```

Comment:
This target is a "integration target" so we don't really "send" stuff to Tekton, but we rather process tasks in the shape of events.

Maybe we should rephrase this slightly to "receive CloudEvents that represent tasks to trigger Tekton builds."?

Reply:
How about instead:
`This target receives a [CloudEvents][ce] over HTTP and will use it to create a [Tekton][tekton] TaskRun or PipelineRun object.`

Reply:
Sounds much better than my clumsy suggestion!